acts-as-list-model-changes

From: Bryan Larsen <bryan@larsen.st>

Now two changes to our models:

SHOW_PATCH
---
 app/models/story.rb |    3 +--
 app/models/task.rb  |    2 ++
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/app/models/story.rb b/app/models/story.rb
index 9e385b8..fd34221 100644
--- a/app/models/story.rb
+++ b/app/models/story.rb
@@ -12,8 +12,7 @@ class Story < ActiveRecord::Base
 
   belongs_to :project, :inverse_of => :stories, :counter_cache => true
   belongs_to :status, :class_name => "StoryStatus", :inverse_of => :stories
-
-  has_many :tasks, :dependent => :destroy, :inverse_of => :story
+  has_many :tasks, :dependent => :destroy, :inverse_of => :story, :order => :position
 
   children :tasks
 
diff --git a/app/models/task.rb b/app/models/task.rb
index dced8e0..1516794 100644
--- a/app/models/task.rb
+++ b/app/models/task.rb
@@ -13,6 +13,8 @@ class Task < ActiveRecord::Base
   has_many :task_assignments, :dependent => :destroy, :inverse_of => :task
   has_many :users, :through => :task_assignments, :accessible => true, :dependent => :destroy
 
+  acts_as_list :scope => :story
+
   # --- Permissions --- #
 
   def create_permitted?
